ansible/roles/install-java/tasks/main.yml

```diff
@@ -15,19 +15,19 @@
 
 ---
 - name: Install Java 11 on RHEL-based systems
-  package:
+  ansible.builtin.package:
```

Comment:
Let's not use the `ansible.builtin.`. I know this is being explicit instead of implicit. However, this is inconsistent with all our other playbooks. Also, this has no impact on the functioning of our playbooks. Let me know of you think otherwise.

Reply:
From what I understand, if the linter suggests we must be explicit then let's be explicit. Also, I think the linter would work on PRs as we make changes to files. So let us not touch all the files and make changes rather with time as we make changes to playbooks, the linter will pop up and state modules explicitly. What do you think?

Reply:
OK, let me close the PR for now